pkg/config/system_probe.go

```diff
@@ -131,6 +133,9 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "zypper_repos_dir"), suffixHostEtc(defaultZypperReposDirSuffix), "DD_ZYPPER_REPOS_DIR")
 	cfg.BindEnvAndSetDefault(join(spNS, "attach_kprobes_with_kprobe_events_abi"), false, "DD_ATTACH_KPROBES_WITH_KPROBE_EVENTS_ABI")
 
+	// User Tracer
+	cfg.BindEnvAndSetDefault(join(diNS, "enabled"), true, "DD_DYNAMIC_INSTRUMENTATION_ENABLED")
```

Comment:
does it make sense to enable the (new) feature by default?

Reply:
Nope - i'll disable